vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -558,7 +555,7 @@ - (void)listOneTimeTips:(void (NS_NOESCAPE ^)(NSArray<BATPublisherInfo *> *))com
 
 - (void)tipPublisherDirectly:(BATPublisherInfo *)publisher amount:(double)amount currency:(NSString *)currency completion:(void (^)(BATResult result))completion
 {
-  ledger->DoDirectTip(std::string(publisher.id.UTF8String), amount, std::string(currency.UTF8String), ^(ledger::Result result) {
+  ledger->DoTip(std::string(publisher.id.UTF8String), amount, nullptr, false, ^(ledger::Result result) {
```

Comment:
What are these extra arguments for? I don't see this used anywhere else since we already have `SaveRecurringTip`  Is there something iOS has to change on our client side?

Reply:
This was done in https://github.com/brave/brave-core/pull/4252/commits/a506beebe7a6d6a2d3ea3426b66b13262d65988f where I just ported what we had before in service here https://github.com/brave/brave-core/pull/4252/commits/a506beebe7a6d6a2d3ea3426b66b13262d65988f#diff-a5d16be9c9b52a6df7b866b1d75d3f5fL2208

Reply:
talked it over DM

Reply:
https://github.com/brave/brave-browser/issues/8403